### DISCUSSION TITLE
Babylon5 support

### DISCUSSION THREAD

**Discussion by jj-cst** [brianzinn/react-babylonjs#203] [Ideas]
Hi! Will there be a react-babylonjs version that supports babylon5? I'm currently trying to use the navigation mesh plugin recast and apparently there is no support currently for babylon 4.2 :( 

Super appreciative of this library!

**Comment by brianzinn** [comment]
Babylon is a peer dependency - is there an error if you are on v5 Babylon?

**Comment by brianzinn** [comment]
I just tested on 5.0 beta.9 - if you have any specific issues please add them.

**Reply by jj-cst** [reply]
Thanks, I will for sure. Currently trying to figure if there is any on my setup; I'm actually debating if I even need to change my version to 5.0 at all. Thanks for finding me on the babylon forum as well haha.

**Comment by brianzinn** [comment]
i'd love a sample demo with recast as well if you get a small example to share - if anything declarative can be added to help out let me know.
